noggin/tests/unit/controller/test_user.py

```diff
@@ -294,3 +294,30 @@ def test_user_can_see_dummy_group(client, dummy_user_as_group_manager):
         )
         == '1 Group Memberships'
     )
+
+
+@pytest.mark.vcr()
+def test_user_settings_agreements(client, logged_in_dummy_user):
+    """Test getting the user agreements page: /user/<username>/settings/agreements/"""
+    result = client.get('/user/dummy/settings/agreements/')
+    page = BeautifulSoup(result.data, 'html.parser')
+    assert page.title
+    assert len(page.select("#agreement-modal-d47df7f4-aabb-11ea-b4d4-525400cc96da")) == 1
```

Comment:
I'm afraid that will fail when we rebuild the cassettes because it depends on your local instance of IPA having the agreements set it. It would be better to add a fixture that would add arguments that we control in the tests, so we know what the IDs are, etc.